cSpell.json

```diff
@@ -0,0 +1,408 @@
+{
+    "version": "0.2",
+    "language": "en",
+    "words": [
```

Comment:
We should add guidelines for adding new valid words to the wiki. It should include where to add the words, what is the distinction between `words` and `ignoreWords` (because it is quite confusing), and a link to the CSpell documentation.

Reply:
I have added it to the readme to of the repository. At first I think we would use spellcheck only for github.io page, however we might discuss the pros and cons also on the other repositories. Then it would be in fact good to have it on wiki